### PR TITLE
fix wrong inSlurm=TRUE message for REMIND-MAgPIE jobs, cleanup

### PR DESCRIPTION
**Pull request by orichters**
- using the old non-parallel mode for REMIND-MAgPIE runs, all finished runs were shown with `inSlurm = TRUE` because it was one single slurm job. With the parallel mode, this is not necessary anymore, so the exception for these runs can be deleted, so they will correctly state their slurm status
- bit of cleanup, such that also `-a` and `-c` uses `loopRuns`, but without color and sorting. Reduces redundancy.

### CODE REVIEW

File: R/foundInSlurm.R [hunk 1]
```diff
@@ -11,11 +11,7 @@
 foundInSlurm <- function(mydir = ".", user = NULL) {
 
   if (is.null(user)) user <- Sys.info()[["user"]]
```
Comment (orichters) [inline R/foundInSlurm.R:13]: the exception for coupled runs is not necessary anymore, as their job name is equal to the folder name now.

File: R/loopRuns.R [hunk 2]
```diff
@@ -14,17 +16,19 @@
 #' loopRuns(dir())
 #' }
 #'
-loopRuns <- function(mydir, user = NULL) {
```
Comment (orichters) [inline R/loopRuns.R:17]: added these options such that `-a` and `-c` output still look the same as before

File: R/promptAndRun.R [hunk 3]
```diff
@@ -1,3 +1,19 @@
+#' promptAndRun
```
Comment (orichters) [inline R/promptAndRun.R:1]: added explanation and also export this function such that it can be used from R directly

File: R/promptAndRun.R [hunk 4]
```diff
@@ -46,23 +62,22 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
     if (length(myruns) == 0) {
       return(paste0("No runs found for this user. You can change the reporting period (here: 5 days) by running 'rs2 -c ", user, " 5"))
     }
+    # add REMIND-MAgPIE coupled runs where run directory is not the output directory
+    # these lines also drops all other slurm jobs such as remind preprocessing etc.
     coupled <- rem <- NULL
     for (i in 1:length(runnames)) {
-      if (any(grepl(runnames[[i]], myruns[[i]]), grepl("mag-run", runnames[[i]]))) {
```
Comment (orichters) [inline R/promptAndRun.R:51]: deleted the non-parallel exception, did a bit of cleanup

File: R/promptAndRun.R [hunk 5]
```diff
@@ -77,27 +91,8 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
       message("Found ", length(myruns), if (mydir == "-a") " active", " runs.",
               if (length(myruns)/as.numeric(daysback) > 20) " Excuse me? You need a cluster only for yourself it seems.")
     }
-
-#    if (length(myruns)>40) {
-#        message("Excuse me? > 40 runs? You need a cluster only for yourself it seems")
-#    } else if (length(myruns)>15) {
-#        message("Please wait, I found more than 15 runs (wow)")
```
Comment (orichters) [inline R/promptAndRun.R:84]: this loop is exactly what `loopRuns` does, just without the color and with different sorting. To reduce redundancy, I added the color = FALSE and sortbytime = FALSE option to loopRuns and can therefore delete most of the stuff here.

File: vignettes/rs2.Rmd [hunk 6]
```diff
@@ -17,28 +17,27 @@ The `rs2` tool gives you quick, easy, and handy access to a number of info and s
 
 ## 5 common use cases:
```
Comment (orichters) [inline vignettes/rs2.Rmd:19]: update to reflect current status of rs2

**Review by pweigmann — APPROVED** [review]
Looks good to me

**Pull request merged**